### PR TITLE
Whats new popup design fixes

### PR DESCRIPTION
**Pull request by danjm**
Makes some tweaks to the Whats New popup UI. Based on feedback from Thomas and a convo with Rachel:

https://user-images.githubusercontent.com/7499938/116746551-39570880-a9d7-11eb-8be3-46115b18e43d.mp4

### CODE REVIEW

File: ui/components/app/whats-new-popup/whats-new-popup.js [hunk 1]
```diff
@@ -40,21 +48,15 @@ const renderFirstNotification = (notification, idRefMap) => {
...
     >
-      {image && (
-        <img
-          className="whats-new-popup__notification-image"
-          src={image.src}
-          height={image.height}
-          width={image.width}
-        />
-      )}
+      {image && !image.placeBelowDescription && imageComponent}
```
Comment (ryanml) [inline ui/components/app/whats-new-popup/whats-new-popup.js:51]: The leading check for `image` shouldn't be necessary here given that it's already checked for here: https://github.com/MetaMask/metamask-extension/pull/10964/files#diff-8a1b237ca446aed86e9d755decb615e0f1b7b3034bf4920781bc494a81aff657R35

**Review by ryanml — CHANGES_REQUESTED** [review]
left one comment, else lgtm!

File: ui/components/app/whats-new-popup/whats-new-popup.js (hunk 1 above)
Comment (danjm) [reply ui/components/app/whats-new-popup/whats-new-popup.js]: Addressed in f1579b911

**Review by ryanml — DISMISSED** [review]

**Review by rachelcope — DISMISSED** [review]
Looks good to me!

**Comment by rachelcope** [comment]
Small thing I noticed. The other modals have 16px padding. Looks like this is 24px padding. I think it would be worth updating to 16px to give the content in the modal a little more room 

image

**Comment by danjm** [comment]
Addressed design feedback: 
Screenshot from 2021-05-03 14-40-03

**Comment by rachelcope** [comment]
This looks good to me!

File: ui/components/app/whats-new-popup/whats-new-popup.js [hunk 2]
```diff
@@ -40,21 +49,15 @@ const renderFirstNotification = (notification, idRefMap) => {
...
-      )}
+      {!placeImageBelowDescription && imageComponent ? imageComponent : null}
       <div className="whats-new-popup__notification-title">{title}</div>
       <div className="whats-new-popup__description-and-date">
         <div className="whats-new-popup__notification-description">
           {description}
         </div>
         <div className="whats-new-popup__notification-date">{date}</div>
       </div>
+      {imageComponent && placeImageBelowDescription ? imageComponent : null}
```
Comment (ryanml) [inline ui/components/app/whats-new-popup/whats-new-popup.js:60]: Could this be refactored to `{placeImageBelowDescription && imageComponent}`?

File: ui/components/app/whats-new-popup/whats-new-popup.js [hunk 3]
```diff
@@ -40,21 +49,15 @@ const renderFirstNotification = (notification, idRefMap) => {
...
     >
-      {image && (
-        <img
-          className="whats-new-popup__notification-image"
-          src={image.src}
-          height={image.height}
-          width={image.width}
-        />
-      )}
+      {!placeImageBelowDescription && imageComponent ? imageComponent : null}
```
Comment (ryanml) [inline ui/components/app/whats-new-popup/whats-new-popup.js:52]: Could this be refactored to `{!placeImageBelowDescription && imageComponent}`?

File: ui/components/app/whats-new-popup/whats-new-popup.js (hunk 2 above)
Comment (danjm) [reply ui/components/app/whats-new-popup/whats-new-popup.js]: yes, thanks, refactored and pushed 7c8fe41da

File: ui/components/app/whats-new-popup/whats-new-popup.js (hunk 3 above)
Comment (danjm) [reply ui/components/app/whats-new-popup/whats-new-popup.js]: yes, thanks, refactored and pushed 7c8fe41da

**Review by ryanml — APPROVED** [review]

**Pull request merged**